.github/workflows/makefile.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
       - name: Install dependencies (pip)
-        run: pip install --requirement requirements.txt --upgrade --upgrade-strategy eager
```

Comment:
Looking at the CI errors, my 1st suspicion would be that each CI step runs in a new shell and therefore the venv is not active for the `make test` step.

Reply:
Fixed that in #150 by using:
```
      - name: Test
        run: |
          . venv
          make test
```